src/main/osd/osd_elements.c

```diff
@@ -945,15 +945,27 @@ static void osdBackgroundHorizonSidebars(osdElementParms_t *element)
 static void osdElementLinkQuality(osdElementParms_t *element)
 {
     uint16_t osdLinkQuality = 0;
-    if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) { // 0-300
-        osdLinkQuality = rxGetLinkQuality()  / 3.41;
-        tfp_sprintf(element->buff, "%c%3d", SYM_LINK_QUALITY, osdLinkQuality);
-    } else { // 0-9
+    uint8_t osdRFmode = 0;
+    if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF)
+    { // 0-300
+        osdLinkQuality = rxGetLinkQuality() / 3.41;
+        osdRFmode = rxGetRFmode();
+        if (osdLinkQuality > 99)
+        {
+            tfp_sprintf(element->buff, "%c%1d:%3d%%", SYM_LINK_QUALITY, osdRFmode, osdLinkQuality);
+        }
+        else
+        {
+            tfp_sprintf(element->buff, "%c%1d:%2d%%", SYM_LINK_QUALITY, osdRFmode, osdLinkQuality);
+        }
+    }
+    else
+    { // 0-9
         osdLinkQuality = rxGetLinkQuality() * 10 / LINK_QUALITY_MAX_VALUE;
         if (osdLinkQuality >= 10) {
             osdLinkQuality = 9;
         }
-        tfp_sprintf(element->buff, "%c%1d", SYM_LINK_QUALITY, osdLinkQuality);
+        tfp_sprintf(element->buff, "%c%1d%%", SYM_LINK_QUALITY, osdLinkQuality);
```

Comment:
I don't think we should add the "%" symbol - particularly here. This is a synthetic value ranging from 0-9. Probably should leave off the "%" on the CRSF version as well since people are used to it this way and we don't use the symbol for other elements like RSSI.

Reply:
Agreed